pallets/gear/src/tests.rs

```diff
@@ -639,10 +644,10 @@ fn delayed_send_user_message_with_reservation() {
 
         run_to_next_block(None);
 
-        assert_eq!(Balances::reserved_balance(&USER_1), 0);
+        // TODO: deal with reserve_for in reserve
```

Comment:
open actual issue on GitHub with explanation and ref the issue here

Reply:
https://github.com/gear-tech/gear/issues/2270